.github/workflows/genlocale.yml

```diff
@@ -13,8 +13,8 @@ jobs:
 
       - name: Run locale generation
         run: |
-          python3 lib/i18n/scan_i18n.py
-          cd lib/i18n && python3 locale_diff.py
+          python3 i18n/scan_i18n.py
+          python3 i18n/locale_diff.py
```

Comment:
Just use `cd` and keep a reletive path in locale_diff.py to adapt to possible future path chages.